ui/components/ui/token-list-display/token-list-display.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { isEqual } from 'lodash';
+
+import { getShouldHideZeroBalanceTokens } from '../../../selectors';
+import { useTokenTracker } from '../../../hooks/useTokenTracker';
+import Identicon from '../identicon';
+import TokenBalance from '../token-balance';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getTokens } from '../../../ducks/metamask/metamask';
+
+export default function TokenListDisplay(props) {
+  const t = useI18nContext();
```

Comment:
[Q] This might be more of a `app/` component? What do you think @georgewrmarshall

Reply:
Yes, I agree this is more suited to an `app/` component